src/data/selectors/index.js

```diff
@@ -227,8 +226,7 @@ export const shouldShowSpinner = (state) => (
  * @return {bool} - should show bulk management controls?
  */
 export const showBulkManagement = (state) => (
-  special.hasSpecialBulkManagementAccess(app.courseId(state))
-  || (tracks.stateHasMastersTrack(state) && state.config.bulkManagementAvailable)
+  tracks.stateHasMastersTrack(state) && state.config.bulkManagementAvailable
```

Comment:
I thought that we were moving the "track" check to the LMS and we were only checking state.config.bulkManagementAvailable? Was this changed?

Reply:
Yeah,  this is duplicate work, already available in a different PR.